i18n(zh-cn): translate `\guides\cms\buttercms.mdx` and `\guides\cms\cloudcannon.mdx` (#3859)
withastro/docs: src/content/docs/zh-cn/guides/cms/cloudcannon.mdx

---
title: CloudCannon与Astro

> description: 使用CloudCannon作为CMS为你的Astro项目添加内容

Reviewer: ```suggestion
description: 使用 CloudCannon 作为 CMS 为你的 Astro 项目添加内容
```

type: cms
stub: true
service: CloudCannon
i18nReady: true
---

> [CloudCannon](https://cloudcannon.com)是一个基于Git的内容管理系统，为你的内容提供了一个可视化的编辑器。

Reviewer: ```suggestion
[CloudCannon](https://cloudcannon.com) 是一个基于 Git 的内容管理系统，为你的内容提供了一个可视化的编辑器。
```

## 官方资源

> - [Astro入门指南](https://cloudcannon.com/documentation/guides/astro-starter-guide/)

Reviewer: ```suggestion
- [Astro 入门指南](https://cloudcannon.com/documentation/guides/astro-starter-guide/)
```

> - [Bookshop & Astro指南](https://cloudcannon.com/documentation/guides/bookshop-astro-guide/)

Reviewer: ```suggestion
- [Bookshop & Astro 指南](https://cloudcannon.com/documentation/guides/bookshop-astro-guide/)
```

> - [Astro初级教程系列](https://cloudcannon.com/tutorials/astro-beginners-tutorial-series/)

Reviewer: ```suggestion
- [Astro 初级教程系列](https://cloudcannon.com/tutorials/astro-beginners-tutorial-series/)
```

> - 博客: [CloudCannon的实时编辑如何与Astro和Bookshop一起工作](https://cloudcannon.com/blog/how-cloudcannons-live-editing-works-with-astro-and-bookshop/)

Reviewer: ```suggestion
- 博客: [CloudCannon 的实时编辑如何与 Astro 和 Bookshop 一起工作](https://cloudcannon.com/blog/how-cloudcannons-live-editing-works-with-astro-and-bookshop/)
```

> - 博客: [为所有Astro用户提供超凡的支持](https://cloudcannon.com/blog/out-of-this-world-support-for-all-astro-users/)

Reviewer: ```suggestion
- 博客: [为所有 Astro 用户提供超凡的支持](https://cloudcannon.com/blog/out-of-this-world-support-for-all-astro-users/)
```
